<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextPropertiesMenu</class>
 <widget class="QWidget" name="TextPropertiesMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>230</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Dubai</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>262</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_title">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Dubai Medium</family>
        <pointsize>9</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Text Properties</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_color">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Medium</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Color</string>
          </property>
          <property name="buddy">
           <cstring>comboBox_color</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_font">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Medium</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Font</string>
          </property>
          <property name="buddy">
           <cstring>comboBox_font</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_size">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Medium</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Size</string>
          </property>
          <property name="buddy">
           <cstring>spinBox_size</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_align">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Medium</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Alignment</string>
          </property>
          <property name="buddy">
           <cstring>comboBox_align</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_style">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Medium</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Style</string>
          </property>
          <property name="buddy">
           <cstring>comboBox_style</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_weight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Medium</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Weight</string>
          </property>
          <property name="buddy">
           <cstring>comboBox_weight</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QComboBox" name="comboBox_color">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Light</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>black</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>white</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>green</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cyan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>magenta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>yellow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>gray</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="comboBox_font">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Light</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="currentText">
           <string>Arial</string>
          </property>
          <property name="currentFont">
           <font>
            <family>Arial</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_size">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Light</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_align">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Light</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>AlignLeft</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AlignRight</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AlignTop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AlignBottom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AlignCenter</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_style">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Light</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>StyleNormal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>StyleItalic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>StyleOblique</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_weight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>190</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Dubai Light</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Thin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Light</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bold</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
